edx_sga/sga.py

```diff
@@ -36,13 +36,16 @@
 
 
 log = logging.getLogger(__name__)
-
+BLOCK_SIZE = 8 * 1024
 
 def reify(meth):
     """
-    Property which caches value so it is only computed once.
```

Comment:
You should change the word `Property` to `Decorator` in the original doc string and just add the parameter description to it instead of removing the short description.